site/gdocs/ProminentLink.tsx

```diff
@@ -50,7 +50,7 @@ export const ProminentLink = (props: {
         description = description ?? linkedDocument?.content.excerpt
```

Comment:
Can't comment on the `href = ...` line above, sadly, but there we probably also want to include query params and hash if they're present?